#!/usr/bin/python3
def raise_exception_msg(message=""):
    a = NameError(message)
    raise a
